src/components/nodes/image_processing/Info.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div/>
+</template>
+
+<script>
+import { BackendImage } from '../../../utils/ajax'
+
+export default {
+  spec: {
+    id: 'std::info',
+    title: 'Image Info',
+    icon: require('../../../assets/node_icons/node_image_info.svg'),
+    inputs: [
+      {
+        id: 'image',
+        name: 'Image',
+        type: 'image'
+      }
+    ],
+    outputs: [
+      {
+        id: 'mean',
+        name: 'Mean',
+        type: 'number'
+      },
+      {
+        id: 'median',
+        name: 'Median',
+        type: 'number'
+      },
+      {
+        id: 'std',
+        name: 'Std. Dev.',
+        type: 'number'
+      }
+    ],
+    async calculate (input) {
+      if (input.image instanceof BackendImage) {
+        const { mean, median, std } = await input.image.info()
+        return { mean, median, std }
```

Comment:
Could this be written as follows?:
```javascript
return ({ mean, median, std } = await input.image.info())
```

Reply:
Unfortunately I don't believe so.

Reply:
Shenanigans.

Reply:
What if you put the `const` keyword in as well?

Reply:
I think a `const` declaration does not return its variables as an object (if anything even). The syntax used here is confusing because it looks like an object instantiation, however it is a structural unpacking of such to export certain fields as variables.
Some sort of object filtering system could be used, and there are libraries for this, but I don't think a dependency or some filtering function just for this would be worth it.

However...
Just returning the entire info object as is would work, since the execution engine will only pass on outputs that are actually defined in the spec, so anything else would just be ignored, but I deemed this option as somewhat ugly and unclean.

Reply:
I understand, thank you for clarifying.

Reply:
You are most certainly very welcome, my good sir.